mdpow/fep.py

```diff
@@ -1309,11 +1309,25 @@ def p_transfer(G1, G2, **kwargs):
            force rereading of data files even if some data were already stored [False]
        *stride*
            analyze every *stride*-th datapoint in the dV/dlambda files
+       *start*
+           Start frame of data analyzed in every fep window.
+       *stop*
+           Stop frame of data analyzed in every fep window.
+       *SI*
+           Set to ``True`` if you want to perform statistical inefficiency
+           to preprocess the data.
+       *estimator*
+           Set to ``alchemlyb`` if you want to use alchemlyb estimators,
+           or ``mdpow`` if you want the default TI method.
+       *method*
+           Use `TI`, `BAR` or `MBAR` method in `alchemlyb`, or `TI` in `mdpow`.
+
     :Returns: (transfer free energy, log10 of the water-octanol partition coefficient = log Pow)
 
     """
 
     kwargs.setdefault('force', False)
+    estimator = kwargs.pop('estimator', 'mdpow')
```

Comment:
Check that `estimator` is one of the supported ones, then we don't need to check below (and you can ignore my comments below)
```suggestion
    estimator = kwargs.pop('estimator', 'mdpow')
    if not estimator in ('mdpow', 'alchemlyb'):
        errmsg = "estimator = %r is not supported, must be 'mdpow' or 'alchemlyb'" % estimator
        logger.error(errmsg)
        raise ValueError(errmsg)
```